pkg/oc/cli/idle/idle.go

```diff
@@ -417,7 +417,7 @@ func findScalableResourcesForEndpoints(endpoints *corev1.Endpoints, getPod func(
 	controllerRefs := make(map[namespacedCrossGroupObjectReference]struct{})
 	for controllerRef := range immediateControllerRefs {
 		controller, err := getController(controllerRef)
-		if utilerrors.TolerateNotFoundError(err) != nil {
+		if err != nil && errors.IsNotFound(err) {
```

Comment:
@mfojtik, I think you lost a negation: 
```
 		if err != nil && !errors.IsNotFound(err) {
```
I noticed this while reviewing https://github.com/openshift/oc/pull/1026.